internal/app/export/service.go

```diff
@@ -177,12 +180,9 @@ func (e *exporter) readMessages(streams []string) (map[string]messages.Msg, erro
 
 func (e *exporter) Subscribe(nc *nats.Conn) {
 	for _, r := range e.consumers {
-		_, err := nc.QueueSubscribe(r.NatsTopic(), e.id, r.Consume)
-		if err == nil {
-			e.logger.Info(fmt.Sprintf("Subscribed to NATS %s", r.NatsTopic()))
-			continue
+		if err := r.Subscribe(nc); err != nil {
```

Comment:
QueueSubsciribe is conceptually different from Subscribe. Are you sure that you need `Subscribe` (topics are different for each service)?

Reply:
each route has its own nats topic to which subscribes

Reply:
Why would `QueueSubscribe()` not permit this? On the contrary - I think we need `QueueSubscribe()` here to assure scalability when Export service is scaled out (when used in the cloud).

Reply:
Please do not resolve this remark - it is not resolved. We need NATS LB - i.e. Queue here.